### ISSUE TITLE
cdb-ext url crashes local native client on startup if not nullified

### ISSUE THREAD

**Issue by SEsternO** [nroduit/weasis-pacs-connector#45]
Using minimal dcm4chee-arc-light 5.29.0, deployed with docker-compose and weasis-pacs-connector 7.1.2.

```
IID_PATIENT_URL=../../weasis-pacs-connector/weasis?patientID={{patientID}}&cdb
IID_STUDY_URL=../../weasis-pacs-connector/weasis?studyUID={{studyUID}}&cdb
```
by the docs.

The problem is in this url that the connector gives to the browser:
`weasis://$dicom:get -w "http://192.168.94.35:8080/weasis-pacs-connector/RequestManifest;jsessionid=Iwd6k2ilkvJgIHqmACNhyOq71aYvjDWq9dLpo2oH.bc985330fa84?id=2" $weasis:config  cdb cdb-ext="http://192.168.94.35:8080/weasis-ext"`,
particularly in the presence of cdb-ext parameter. With it, the local native client crashes with "Failed to launch JVM" error.

I didn't manage to find anything about it in the docs (mostly why I type this issue), but i did a workaround myself, as it's similar to nullifying the cdb parameter - just nullify cdb-ext as well, e.g.:
```
IID_PATIENT_URL=../../weasis-pacs-connector/weasis?patientID={{patientID}}&cdb&cdb-ext
IID_STUDY_URL=../../weasis-pacs-connector/weasis?studyUID={{studyUID}}&cdb&cdb-ext
``` 
and the client starts and opens images as it should.

That said, besides weasis-pacs-connector I also first installed weasis.war and weasis-ext.war, but then saw that weasis.war is discontinued and deleted both weasis.war and weasis-ext.war. Not sure if that might've messed up the things in the first place somehow.

**Comment by nroduit** [comment]
Yes Weasis server side is not mandatory, it allows to update minor versions of Weasis without the need to update Weasis with the installer.

We will soon propose a new component that will replace weasis-pacs-connector and allow to better manage user preferences and launch contexts.
